frontend/public/components/events.jsx

```diff
@@ -70,7 +70,16 @@ const Inner = connectToFlags(FLAGS.CAN_LIST_NODE)(class Inner extends React.Pure
               name={obj.name}
               title={obj.uid}
             />
-            <Timestamp timestamp={lastTimestamp} />
+            {obj.namespace
+              ? <ResourceLink
+                className="co-sysevent__resourcelink co-sysevent__namespacelink"
+                kind="Namespace"
+                namespace={obj.namespace}
+                name={obj.namespace}
+                title={obj.namespace}
+              />
+              : <span className="co-sysevent__resourcelink co-sysevent__namespacelink">all</span>}
```

Comment:
I'm starting to think we should leave off "all" in this case. The difference between this list and role bindings is that role bindings do apply to all namespaces, whereas this event just applies to a cluster-scoped resource and is not relevant for other namespaces.

We should revisit the "all" in the cluster resource quota page as well since cluster resource quotas only affect projects that match a label selector, not all projects. I think it might be misleading there as well.

Reply:
Added issue [CONSOLE-1225](https://jira.coreos.com/browse/CONSOLE-1225) in JIRA to address cluster resource quota page.